x/gov/client/testutil/deposits.go

```diff
@@ -4,13 +4,14 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/stretchr/testify/suite"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
```

Comment:
while these changes are correct, it makes revieweing hard (82 files changed). If possible, in the future, let's avoid making side-changes in a big PR, prefer to have 1 PR that's only focused on fixing lints

Reply:
I do not get what you mean here. Could you clarify, please?